frontend/plan/src/components/selector/Section.js

```diff
@@ -1,40 +1,77 @@
+        <div className="course_section">
+            <li style={{
+                display: "flex", alignItems: "center", marginBottom: "0", borderBottom: "1px solid rgb(230, 230, 230)",
+            }}
+            >
+                <div style={{
+                    display: "flex", flexDirection: "column", flexGrow: "1", padding: "1em", paddingRight: "0",
+                }}
                 >
-                    <div
-                        role="button"
-                        onClick={inCart ? cart.remove : cart.add}
-                    >
-                        {inCart ? (
-                            <div className="hover-switch">
-                                <i className="fas fa-check" />
-                                <i className="fas fa-times" />
-                            </div>
-                        ) : <i className="fas fa-plus" />}
+                    <div style={{ display: "flex", justifyContent: "space-between", alignItems: "center" }}>
+                        <div style={{ fontSize: "1rem", fontWeight: "bold", marginRight: "1em" }}>
+                            {section.id.split("-").pop()}
+                        </div>
+                        <div style={{
+                            fontSize: "0.85rem", fontWeight: "bold", textOverflow: "ellipsis", flexGrow: "6",
+                        }}
+                        >
+                            {instructors.length > 0
+                                ? (
+                                    <>
+                                        {instructors.map((elem, ind) => (
+                                            <>
+                                                {ind !== 0 ? <br /> : null}
+                                                {ind !== instructors.length - 1 ? `${elem},` : elem}
+                                            </>
+                                        ))}
+                                    </>
+                                )
+                                : <div> N/A </div>
+                            }
+                        </div>
                     </div>
-                    <div>
-                        {section.id.split("-").pop()}
+                    <div style={{
+                        marginTop: "0.5em", display: "grid", fontSize: "0.7rem", gridTemplateColumns: "40% 15% 40%",
+                    }}
+                    >
+                        <div>
+                            {getTimeString(meetings)}
+                        </div>
+                        <div>
+                            {section.activity}
+                        </div>
+                        <div>
+                            {meetings.length > 0 ? (
+                                <div>
+                                    <i className="fas fa-map-marker-alt grey-text" />
+                            &nbsp;
+                                    {((l) => {
+                                        const ret = new Set();
```

Comment:
A comment about what this code is doing (formatting the location info) would be helpful